dhcpv6/server6/server_test.go

```diff
@@ -53,6 +60,29 @@ func setUpClientAndServer(handler Handler) (*nclient6.Client, *Server) {
 	return c, s
 }
 
+type customLogger struct {
+	called bool
+	mux    sync.Mutex
+}
+
+func (s *customLogger) Printf(format string, v ...interface{}) {
+	s.mux.Lock()
+	s.called = true
+	s.mux.Unlock()
+	log.Println("===CustomLogger BEGIN===")
```

Comment:
since this is in a test, do

```go
type customLogger struct {
 tb testing.TB
 called bool
  mux sync.Mutex
}

func (s *cus ... {
  s.TB.Logf(...)
```

Reply:
it looks like the testing package log is not thread-safe, though. It's failing in Travis

Reply:
@pmazzini @hugelgupf since the log function from the testing package is not thread-safe, do you have any suggestions to overcome this issue?

Reply:
Is it not thread safe? It is strange it is only failing on Go 1.12. I need to take a look.

Reply:
The log function from the testing package looks to be protected by a lock. I am not sure where this race is coming from.

Reply:
I removed the server specific tests because it was not adding much other than checking that the logger was called. I did it so that we can get this unblocked.